crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -72,10 +72,10 @@ pub fn mk_add_card_request(
     _merchant_id: &str,
 ) -> CustomResult<services::Request, errors::CardVaultError> {
     #[cfg(feature = "sandbox")]
-    let customer_id = format!("{}::{}", customer_id, _merchant_id);
+    let customer_id = &format!("{}::{}", customer_id, _merchant_id);
```

Comment:
please, rename `_merchant_id` -> `merchant_id`

Reply:
The checks fail on CI